server/src/main/java/umm3601/user/User.java

```diff
@@ -34,4 +34,19 @@ public class User {
   public String email;
   public String avatar;
   public String role;
+
+  @Override
+  public boolean equals(Object obj) {
+    if (!(obj instanceof User)) {
+      return false;
+    }
+    User other = (User) obj;
+    return _id.equals(other._id);
+  }
+
+  @Override
```

Comment:
Good catch on the need to override the `hashCode` – sorry I missed that.

Reply:
No worries!